src/main/resources/view/MainWindow.fxml

```diff
@@ -29,7 +29,9 @@
             <MenuItem mnemonicParsing="false" onAction="#handleExit" text="Exit" />
           </Menu>
           <Menu mnemonicParsing="false" text="Help">
-            <MenuItem fx:id="helpMenuItem" mnemonicParsing="false" onAction="#handleHelp" text="Help" />
+            <MenuItem fx:id="helpMenuItem" styleClass="link" text="CareConnect User Guide on Web"
```

Comment:
There seem to be a UI issue where the help window shows some extra text
<img width="747" alt="image" src="https://github.com/user-attachments/assets/216064ac-4e57-4fc9-8952-5dd42cadd16c">

Reply:
Ah I see it's a keyboard shortcut feature